eslint.config.mjs

```diff
@@ -0,0 +1,10 @@
+import globals from 'globals'
+import pluginJs from '@eslint/js'
+import tseslint from 'typescript-eslint'
```

Comment:
is typescrpit lint needs are the code should be typescript?

Reply:
sdk is written without adding eslint to the workspace... If you want I can remove it.